### ISSUE TITLE
Seeking Assistance with Errors in Example Run

### ISSUE THREAD

**Issue by yanjin-tech** [SMTG-Bham/doped#67]
Hello,

I seem to have encountered some errors while running an example. Could you please advise on how to proceed?

Traceback (most recent call last):
  File "/public/home/jyan/test.py", line 8, in <module>
    cp = CompetingPhases("ZrO2")
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/public/home/jyan/anaconda3/lib/python3.11/site-packages/doped/chemical_potentials.py", line 395, in __init__
    self.MP_full_pd_entries = mpr.get_entries_in_chemsys(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/public/home/jyan/anaconda3/lib/python3.11/site-packages/mp_api/client/mprester.py", line 1192, in get_entries_in_chemsys
    self.get_entries(
  File "/public/home/jyan/anaconda3/lib/python3.11/site-packages/mp_api/client/mprester.py", line 759, in get_entries
    doc.model_dump()[property]  # type: ignore
    ~~~~~~~~~~~~~~~~^^^^^^^^^^
**KeyError: 'pretty_formula'**

Thank you for your time and assistance.

**Comment by adair-nicolson** [comment]
Hi @yanjin-tech 

Are you using the new Materials Project API, rather than the legacy API? ```Doped``` currently only supports the old legacy API, see installation [docs.](https://doped.readthedocs.io/en/latest/Installation.html#setup-potcars-and-materials-project-api) This issue has been raised in the [past](https://github.com/SMTG-Bham/doped/issues/57#issuecomment-2023009849) and the response goes into detail explaining why we still use the legacy API. 

Hope this helps!

**Comment by kavanase** [comment]
As @adair-nicolson points out, this error is caused by using the new Materials Project API, for which `doped` is currently not compatible (though will be updated to also work with this in the near future). For now, I've updated our error catching to give a more informative error message when this issue occurs (https://github.com/SMTG-Bham/doped/commit/7577c5fd502fb3bbdcb5605025112154e3c092c0). 
Closing this issue now as the problem has been addressed.
